From 9f93eac4fd85544997c163c40e16561f5cce8398 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Fri, 17 Nov 2017 10:59:15 +0800
Subject: [PATCH 2830/5242] MLK-16837: ARM64: dts: fix the noise isse with
 mono stream

commit  e8e277e94d999544adbf4a8c121d49083fb6c614 from
https://source.codeaurora.org/external/imx/linux-imx.git

When play the mono stream, there is no data in right channel,
but if config the pin IO in pull up or pull down state, the codec
can get no zero data in right channel, then user can hear noise.
config the pin in no pull state to fix the noise issue.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts   |    2 +-
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts |    2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts   |    2 +-
 .../freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts   |    2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |    2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
index 4c26ccd..9b5a598 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-mqs.dts
@@ -124,7 +124,7 @@
 
 		pinctrl_sai0: sai0grp {
 			fsl,pins = <
-				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600004c
+				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600006c
 				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc600004c
 				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc600004c
 				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc600004c
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
index 15eb96f..db8a047 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-spdif.dts
@@ -123,7 +123,7 @@
 
 		pinctrl_sai0: sai0grp {
 			fsl,pins = <
-				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600004c
+				SC_P_SAI1_RXC_AUD_SAI0_TXD		0xc600006c
 				SC_P_SAI1_RXFS_AUD_SAI0_RXD		0xc600004c
 				SC_P_SAI1_TXC_AUD_SAI0_TXC		0xc600004c
 				SC_P_SPI2_CS1_AUD_SAI0_TXFS		0xc600004c
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 11f6e97..72d340f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -287,7 +287,7 @@
 				SC_P_SAI1_RXD_AUD_SAI1_RXD		0x0600004c
 				SC_P_SAI1_RXC_AUD_SAI1_RXC		0x0600004c
 				SC_P_SAI1_RXFS_AUD_SAI1_RXFS		0x0600004c
-				SC_P_SAI1_TXD_AUD_SAI1_TXD		0x0600004c
+				SC_P_SAI1_TXD_AUD_SAI1_TXD		0x0600006c
 				SC_P_SAI1_TXC_AUD_SAI1_TXC		0x0600004c
 			>;
 		};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
index b441fcc..57fef5f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2-wm8962.dts
@@ -64,7 +64,7 @@
 			fsl,pins = <
 				SC_P_SAI0_TXFS_ADMA_SAI0_TXFS		0xc600004c
 				SC_P_SAI0_TXC_ADMA_SAI0_TXC		0xc600004c
-				SC_P_SAI0_TXD_ADMA_SAI0_TXD		0xc600004c
+				SC_P_SAI0_TXD_ADMA_SAI0_TXD		0xc600006c
 				SC_P_SAI0_RXD_ADMA_SAI0_RXD		0xc600004c
 				SC_P_SPI2_SDO_LSIO_GPIO1_IO01		0xc600004c
 				SC_P_SPI2_SDI_LSIO_GPIO1_IO02		0xc600004c
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 2bc2d80..15fe65c 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -327,7 +327,7 @@
 				SC_P_SAI1_RXD_ADMA_SAI1_RXD	0x0600004c
 				SC_P_SAI1_RXC_ADMA_SAI1_TXC	0x0600004c
 				SC_P_SAI1_RXFS_ADMA_SAI1_TXFS	0x0600004c
-				SC_P_SPI0_CS1_ADMA_SAI1_TXD	0x0600004c
+				SC_P_SPI0_CS1_ADMA_SAI1_TXD	0x0600006c
 				SC_P_SPI2_CS0_LSIO_GPIO1_IO00	0x0600004c
 			>;
 		};
-- 
1.7.9.5

